src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -263,6 +271,37 @@ def _create_batch_script(job_dir, task_list_file, n_jobs, tool):
     return batch_script_file
 
 
+def _create_job_dir(root_dir, tool, kbase_collection, load_ver, force=False):
+    """
+    Create the job directory
+
+    Job directory structure: <root_dir>/<TASKFARMER_JOB_DIR>/<kbase_collection>_<load_ver>_<tool>
+    """
+
+    job_dir = os.path.join(root_dir, TASKFARMER_JOB_DIR, f'{kbase_collection}_{load_ver}_{tool}')
```

Comment:
This doesn't take the source data dir into account, which maybe is ok, but the user probably needs to know that effectively the load version is bound to that source data dir unless they `--force` (which kind of makes sense).

Also I assume this will need adjusting when we start doing subsets of the source data dir, like bacdive or whatever?

Reply:
👍 added `source_data_dir`

Reply:
Ok, so I see `source_data_dir` was added to the job info file, but the directory is still named solely based on the collection, tool, and load_ver. Is the strategy that on running again the code will check the `source_data_dir` in the job info file and warn or error if it's different than the `source_data_dir` in the program args?

Reply:
Also I still have the question about data subsets / bacdive etc. above

Reply:
yes. following PRs will check if the current task has a conflict source_data_dir from the latest run and prompt users to use `-force` option. 

Regarding to subsets, I think each product, bacdive or gtdb, should have its own source data directory. And files within are just links to the NCBI directory.

Reply:
:+1: